### ISSUE TITLE
QStepper emits not set in Quasar V2

### ISSUE THREAD

**Issue by ovidiuc** [quasarframework/quasar#8480]
**Describe the bug**
Emits option not set for QStepper.

**Codepen/jsFiddle/Codesandbox (required)**
The warning does not show up in Codepen for some reason.

**To Reproduce**
`quasar create quasar.stepper-issue-1 --branch next`

```
? Project name (internal usage for dev) quasar.stepper-issue-1
? Project product name (must start with letter if building mobile apps) Quasar App
? Project description A Quasar Framework app
? Author
? Pick your CSS preprocessor: SCSS
? Check the features needed for your project: Vue-i18n
? Continue to install project dependencies after the project has been created? (recommended) yarn
````

Copy into src/pages/Index.vue the code from the first QStepper example (https://next.quasar.dev/vue-components/stepper#Horizontal), including the JS part.

Run `quasar dev`

Navigate to the website in your browser, and, with the development console open, move through the Stepper's steps.

Multiple warnings will show up:

```
[Vue warn]: Component emitted event "transition" but it is neither declared in the emits option nor as an "onTransition" prop.

[Vue warn]: Component emitted event "update:modelValue" but it is neither declared in the emits option nor as an "onUpdate:modelValue" prop.

[Vue warn]: Component emitted event "before-transition" but it is neither declared in the emits option nor as an "onBefore-transition" prop.
```

**Expected behavior**
There should be no event related console warnings.

**Screenshots**

![image](https://user-images.githubusercontent.com/6917631/107953033-35ecdd00-6fa3-11eb-90e9-36c2f243ec22.png)

**Platform (please complete the following information):**
Quasar Version: v2.0.0-beta.4
@quasar/app Version: v3.0.0-beta.6
Quasar mode:
  [* ] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
Tested on:
  [* ] SPA
  [ ] SSR
  [ ] PWA
  [ ] Electron
  [ ] Cordova
  [ ] Capacitor
  [ ] BEX
OS: Debian GNU/Linux 9 (Docker)
Node: v14.15.1
NPM:
Yarn: 1.22.5
Browsers: Chrome Version 88.0.4324.150 (Official Build) (64-bit)
iOS:
Android:
Electron:

**Comment by rstoenescu** [comment]
Thanks for reporting!
Fix will be available in Quasar 2.0.0-beta.5

**Comment by craigrileyuk** [comment]
Getting the same on q-tabs when navigating between panels too.

**Comment by rstoenescu** [comment]
@craigrileyuk You're right. For some reason, I missed that also. But fixed. Will be available in Quasar v2.0.0-beta.8.
